libs/platform/src/lib/components/form/base.input.ts

```diff
@@ -209,6 +209,15 @@ export abstract class BaseInput extends BaseComponent
 
     /** @hidden */
     ngAfterViewInit(): void {
+
+        if (this.ngControl) {
+            this._subscriptions.add(
+                this.ngControl.statusChanges.subscribe(() => {
+                    this.markForCheck();
```

Comment:
I think we should make it conditional. Otherwise will be firing like crazy.

Reply:
It fires only when the previous validation status does not equal the current, so there shouldn't be the case when it's triggered on every user keydown.